docs/internal-documentation/maintenance/ReleaseProcess-release.md

```diff
@@ -267,25 +268,9 @@ Make sure you're up to date and your working directory is completely clean:
 > **Note:** To throw away any and all changes to tracked and untracked files you
 > can run `git clean -dfx ; git reset --hard`.
 
-Create a new branch named `merge-develop-into-master` with both *master* and
-*develop* as ancestors and with the exact contents from *develop*:
+Create a pull request from `develop` into `master` and merge it.
 
-    # We want the contents of the *develop* branch
-    git checkout -b merge-develop-into-master origin/develop
-
-    # But on top of the history of the *master* branch
-    git reset --soft origin/master
-
-    # With all differences squashed into a single commit
-    git commit -m 'Update master to state of develop'
-
-    # And we want the history of *develop* merged too
-    git merge origin/develop
-
-Create a pull request from `merge-develop-into-master` into `master` and merged
-it. No reviewer or approval is required for this update.
-
-In [Appendix B] it is shown how `merge-develop-into-master` can be verified.
+In [Appendix C] it is shown how to rollback if needed.
```

Comment:
Can you explain when this step will be taken in the release process?

Reply:
Removed for now as this is not exactly addressing the problem